precode.sh

```diff
@@ -6,38 +6,40 @@
 #   dir2
 #   dir3
 #       dir4
-
+mkdir -p ./task/dir1 ./task/dir2 ./task/dir3/dir4
```

Comment:
Можно не указывать везде `./`, будет работать так же. Обязательно это нужно только при запуске скрипта на 34 строчке
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```